components/elements/inputs/text-input.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="flex flex-col">
-    <label for="" class="text-mtl-grey-500">{{ label }}</label>
+    <label for class="text-mtl-black-400">{{ label }}</label>
     <input v-bind="$attrs" />
```

Comment:
For accessibility, the label should enclose the input, otherwise you must provide a valid element id

```suggestion
    <label class="text-mtl-black-400">
      {{ label }}
      <input v-bind="$attrs" />
    </label>
```